<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogHTTP</class>
 <widget class="QDialog" name="DialogHTTP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Abrir link HTTP</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="6">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="LabelVideo">
        <property name="text">
         <string>/videofeed</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>http://</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>http://</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QLineEdit" name="LineEditPortAudio">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="LineEditAddressVideo"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="LineEditAddressAudio">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLineEdit" name="LineEditPortVideo"/>
      </item>
      <item row="2" column="9">
       <widget class="QLabel" name="LabelAudio">
        <property name="text">
         <string>/audio.wav</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="CheckBoxHTTPVideo">
        <property name="text">
         <string>Solo Video</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="CheckBoxHTTPAudio">
        <property name="text">
         <string>Solo Audio</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogHTTP</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogHTTP</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
